src/main/java/uk/gov/beis/els/util/FontUtils.java

```diff
@@ -0,0 +1,61 @@
+    );
+
+
+    // Character widths are relative to font size in pixels. For example, if a character is 10px wide when the font
+    // size is 10px, the character width is 1. If it's 7px wide when the font size is 10px, the character width is 0.7
+    private final Map charWidths;

```

Comment:
This should be declared as a `Map<Character, Float>`, then you won't have to cast to Float below (and its typesafe)